src/linear_algebra/basis.lean

```diff
@@ -94,6 +94,8 @@ namespace basis
 
 instance : inhabited (basis ι R (ι →₀ R)) := ⟨basis.of_repr (linear_equiv.refl _ _)⟩
 
+instance [subsingleton R] : unique (basis ι R M) := ⟨⟨⟨default⟩⟩, λ ⟨b⟩, by rw subsingleton.elim b⟩
```

Comment:
I think theres a unique_of_subsingleton constructor

Reply:
Unfortunately, there isn't a `subsingleton` instance around yet. I'm transferring the `inhabited` and `subsingleton` instances from the underlying `linear_equiv` to `basis` in one go.